RoddyCore/RoddyCore.iml

```diff
@@ -14,5 +14,6 @@
     <orderEntry type="library" name="R Skeletons" level="application" />
     <orderEntry type="module" module-name="RoddyToolLib_main" />
     <orderEntry type="module" module-name="BatchEuphoria_main" />
+    <orderEntry type="library" name="roddyBundledLibraries" level="project" />
```

Comment:
I'm not sure. Didn't we agree on a global lib?

Reply:
Sorry for this change. Yes we agreed on project-level library. This change reverts one that sneaked into a previous diff, because I was testing gradle.